### PR TITLE
Feature Policy update: battery, webauthn, publickey-credentials, xr-spatial-tracking, and vr

### PR DESCRIPTION
**Pull request by bershanskiy**
## Summary
Chrome team introduced Feature Policy directive `battery` that controls Battery Status API. This directive is already on standards track, has Web Platform Tests, and Chrome team is implementing it. Firefox and Safari do not plan to implement this feature because Safari never supported Battery Status API and Firefox restricted it to privileged contexts.

`webauthn` directive was renamed `publickey-credentials`.

Related issue: #5095

### `battery`
 - [Chromium bug](https://crbug.com/1007264)
 - [MDN article on Feature-Policy: battery](https://wiki.developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy/battery)
 - [Battery Status API Feature Policy integration](https://w3c.github.io/battery/#feature-policy-integration)
 - [List of standard features](https://github.com/w3c/webappsec-feature-policy/blob/master/features.md#standardized-features) with `battery` identifier

### `webauthn` becomes `publickey-credentials`
 - [Web Authentication API Feature Policy integration](https://w3c.github.io/webauthn/#sctn-feature-policy)
 - [Chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=993007)
 - [List of standard features](https://github.com/w3c/webappsec-feature-policy/blob/master/features.md#standardized-features) with `publickey-credentials` identifier

#### Related issues
[`webauthn` MDN article slug update](https://github.com/mdn/sprints/issues/2441) (fixed)

### `vr` deprecation and introduction of `xr-spatial-tracking`
Since WebVR API was deprecated and eventually will be removed, it makes sense to mark `vr` feature deprecated.
Also, I added `xr-spatial-tracking` feature because it is already standardized and Chromium team works on the implementation. FYI, Chrome 80 Canary already reports it in `document.featurePolicy.features()`.

Related issue: #5094

A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any

### CODE REVIEW

**Comment by bershanskiy** [comment]
FYI, I added both features in one PR because seems most logical to update the whole file in one sweep, but feel free to cherry-pick either commit if you want them separated.

File: http/headers/feature-policy.json [hunk 1]
```diff
@@ -294,6 +294,55 @@
...
+        "battery": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/battery",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "notes": "Will be implemented, see <a href='https://crbug.com/1007264'>bug 1007264</a>."
+              },
+              "chrome_android": {
+                "version_added": false
```
Comment (jpmedley) [inline http/headers/feature-policy.json]: The bug note should be here as well.

Comment (bershanskiy) [reply http/headers/feature-policy.json]: Is there a plan to ship `battery` for Android WebView? If so, I should insert that note too.

Comment (bershanskiy) [reply http/headers/feature-policy.json]: Marked resolved because I added the note to `chrome_android`.

**Comment by jpmedley** [comment]
LGTM for Chrome data.

**Pull request merged**